superset/views/api.py

```diff
@@ -44,7 +44,7 @@ def query(self):
         security_manager.assert_query_context_permission(query_context)
         payload_json = query_context.get_payload()
         return json.dumps(
-            payload_json, default=utils.json_int_dttm_ser, ignore_nan=True
+            payload_json, default=utils.json_int_dttm_ser, allow_nan=False
         )
```

Comment:
Perhaps we should leave this as-is, and only implement `allow_nan` in the new EP to correspond with vanilla `json.dumps`, just in case. Looking at the source code, it seems `ignore_nan` is supposed to take precedence over `allow_nan` in `simplejson`: https://github.com/simplejson/simplejson/blob/288e4e005c39a2eb855b5225c5dc8ebcb82eee72/simplejson/__init__.py#L373-L376

Reply:
I actually made the switch from simplejson to json since it's the more current stdlib. Yet json does not implement `ignore_nan` just `allow_nan`. Was taking the chance to update

Reply:
reverted, this change should not exist